packages/babel-plugin-transform-async-generator-functions/test/fixtures/nested/arrows-in-declaration/expected.js

```diff
@@ -2,9 +2,7 @@ let g = (() => {
   var _ref = babelHelpers.asyncGenerator.wrap(function* () {
     var _this = this;
 
-    (function () {
-      return _this;
-    });
+    () => this;
```

Comment:
These changes are from the removed call on https://github.com/babel/babel/pull/5677/files#diff-3953cdd7a44c58e0431a2e202efb64d4L31 that wasn't needed.